app/controllers/items_controller.rb

```diff
@@ -10,15 +11,44 @@ def index
     end
   end
 
+  def new
+    @item = current_business.items.new
+  end
+
   def create
-    # TODO: [MOKA-001] Finish the implementation of Create/Update/Delete for Items
+    @item = current_business.items.new(item_params)
+    if @item.save
+      redirect_to @item, notice: 'item was successfully created.'
```

Comment:
YAGNI, there is no need to have i18n right now, you don't have language switcher yet.